### ISSUE TITLE
invalid face: org-level-1

### ISSUE THREAD

**Issue by reverseila** [daviwil/emacs-from-scratch#34]
``` lisp
(dolist (face '((org-level-1 . 1.2)
                (org-level-2 . 1.1)
                (org-level-3 . 1.05)
                (org-level-4 . 1.0)
                (org-level-5 . 1.1)
                (org-level-6 . 1.1)
                (org-level-7 . 1.1)
                (org-level-8 . 1.1)))
  (set-face-attribute (car face) nil :font "Cantarell" :weight 'regular :height (cdr face)))
```
when I evaluate this code, emacs doesn't complain but when I load a new instance of emacs it gives me an error of: `error: Invalid face, org-level-1`.

emacs version : 27.1, org version: 9.3

**Comment by reverseila** [comment]
Thanks to lawlist on emacs stack exchange we have an [solution](https://emacs.stackexchange.com/questions/62981/error-invalid-face-org-level-1/62986#62986) for the problems like this. As I'm new to emacs I just fixed it with doing:
```
(with-eval-after-load 'org-faces
;; Increase the size of various headings
(set-face-attribute 'org-document-title nil :font "Cantarell" :weight 'bold :height 1.3)
(dolist (face '((org-level-1 . 1.2)
                (org-level-2 . 1.1)
                (org-level-3 . 1.05)
                (org-level-4 . 1.0)
                (org-level-5 . 1.1)
                (org-level-6 . 1.1)
                (org-level-7 . 1.1)
                (org-level-8 . 1.1)))
  (set-face-attribute (car face) nil :font "Cantarell" :weight 'regular :height (cdr face)))

;; Ensure that anything that should be fixed-pitch in Org files appears that way
(set-face-attribute 'org-block nil    :foreground nil :inherit 'fixed-pitch)
(set-face-attribute 'org-table nil    :inherit 'fixed-pitch)
(set-face-attribute 'org-formula nil  :inherit 'fixed-pitch)
(set-face-attribute 'org-code nil     :inherit '(shadow fixed-pitch))
(set-face-attribute 'org-table nil    :inherit '(shadow fixed-pitch))
(set-face-attribute 'org-verbatim nil :inherit '(shadow fixed-pitch))
(set-face-attribute 'org-special-keyword nil :inherit '(font-lock-comment-face fixed-pitch))
(set-face-attribute 'org-meta-line nil :inherit '(font-lock-comment-face fixed-pitch))
(set-face-attribute 'org-checkbox nil  :inherit 'fixed-pitch))
```
basically I just wrapped all the "set-face-attributes" that caused me trouble, with `(with-eval-after-load 'org-faces <codes>)`.

**Comment by daviwil** [comment]
Hey @reverseila, thanks for sharing the solution!  Would you be interesting in submitting a PR to this repo to fix it?  If you don't have time, I'll take care of it.

**Comment by reverseila** [comment]
hey @daviwil, sorry for late response. I don't know how to do that. I'm learning git.

**Comment by daviwil** [comment]
No problem, I'll send it.  Thanks again for pointing this out!

**Comment by reverseila** [comment]
my pleasure!

**Comment by Skalas** [comment]
I can do this! :) let me fork this and I'll set up the pull request. (it is still unfixed as I can see)

**Comment by Skalas** [comment]
It is fixed on #53
